### DISCUSSION TITLE
guided_stabilzier is not valid while walking

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#36] [Q&A] [answered]
この処理は論文のどの式に該当しますか?
https://github.com/YutaKojio/hrpsys-base/blob/5aab7f9809b1ef76dd62bf323e522834d84097be/rtc/AutoBalancer/Stabilizer.cpp#L540-L544

**Comment by YutaKojio** [comment, marked as answer]
ないですが，まだpublishしてないのだと https://drive.google.com/file/d/1udUfdh1jdyYmd57A5ereUekfseP369N7/view?usp=sharing の式(20)です．
また，
運足誘導型二脚ロボット制御の可捕性条件緩和による計算安定化, RSJ'20
の式(10)あたりにより詳細な議論が書いてあります．
